src/dto/ReadLoanDto.ts

```diff
@@ -7,5 +7,6 @@ export class LoanDto {
     bookCopyId?: number;
     isActive?: Boolean;
     userId?: number;
-    bookId?: number;
+    bookTitle?: string;
+    bookAuthors?: [];
```

Comment:
ReadLoanDto geeft in de backend nu toch bookId mee, kan hier ook opgeslagen worden. Om via boek title naar bookDetailPagina te kunnen.